### PR TITLE
[New Exercise] Lens person

### PR DESCRIPTION
**Pull request by Cool-Katt**
As discussed in #2400, this PR is for adding the `lense-person` exercise.

Implementation is by @sarava338, I helped with formatting.

---
Closes #2400

### CODE REVIEW

**Comment by Cool-Katt** [comment]
bump?

File: exercises/practice/lens-person/.meta/proof.ci.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-vars */
+import { Person } from '../person';
+import { Name } from '../name';
+import { Born } from '../born';
+import { Address } from '../address';
+import { Lens } from '../lens';
```
Comment (SleeplessByte) [inline exercises/practice/lens-person/.meta/proof.ci.js:6]: Nit: can you order this so we don't have discussions later down the line/people using automatic ordering in vscode.

File: exercises/practice/lens-person/lens-person.spec.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import { Person } from './person';
+import { Name } from './name';
+import { Address } from './address';
+import { Born } from './born';
```
Comment (SleeplessByte) [inline exercises/practice/lens-person/lens-person.spec.js:4]: Same here

File: exercises/practice/lens-person/lens-person.spec.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+// import { nameLens, bornAtLens, streetLens } from './.meta/proof.ci';
```
Comment (SleeplessByte) [inline exercises/practice/lens-person/lens-person.spec.js:6]: Please remove this

File: exercises/practice/lens-person/.meta/config.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    ],
+    "example": [
+      ".meta/proof.ci.js"
+    ],
+    "editor": [
+      "address.js",
+      "born.js",
+      "lens.js",
+      "name.js",
+      "person.js"
```
Comment (SleeplessByte) [inline exercises/practice/lens-person/.meta/config.json:21]: nice

**Review by SleeplessByte — APPROVED** [review]
Tiny nit, then can merge.

**Comment by Cool-Katt** [comment]
Is this any better? @SleeplessByte 
It's been a while since I last touched this, turns out

**Comment by SleeplessByte** [comment]
Yeah. Let's go!

**Pull request merged**